composer.json

```diff
@@ -35,11 +38,19 @@
             "email": "levim@php.net"
```

Comment:
is this a good time to generalise to otel php contributors ?

Reply:
Don't know, what do you excatly mean? Using the same entry as in the other composer files?

Reply:
I was thinking the same as your recent addition to the sub-packages - but it might need more discussion so as not to offend anybody

Reply:
Yeah, that was the reason I did not change anything in the main composer file. Let's just postpone this decision. It's such a minor change. We can throw it in any time, but I guess it would be better to discuss it upfront.